source/content/guides/multisite/06-search-replace.md

```diff
@@ -97,9 +101,31 @@ For subdomain Multisites, environments to be replaced are defined and paired in
 
 1. Commit the  `sites.yml` file in the `private/sites.yml` in the site’s Git repository.
 
-### Subdirectory WPMS
+### Subdomain to Subdirectory Conversion
+Sites can also be configured to use subdomain Multisite on the live environment, and subdirectory in all other instances.
 
-No additional configuration is needed if you already completed the steps in [Enable Search and Replace](/guides/multisite/search-replace/#enable-search-and-replace). Search and replace will match the behavior of the platform’s search and replace for non-WPMS sites.
+This can be configured by making `SUBDOMAIN_INSTALL` conditional in `wp-config.php`,
+
+    ```php:title=wp-config.php
+    if ( ! empty( $_ENV['PANTHEON_ENVIRONMENT'] ) && $_ENV['PANTHEON_ENVIRONMENT'] ==  'live' ) {
+      define( 'SUBDOMAIN_INSTALL', true );
```

Comment:
If `SUBDOMAIN_INSTALL` is true on a site with all subdirectories, does it break anything? I looked through [core](https://github.com/search?q=repo%3AWordPress%2FWordPress%20is_subdomain_install&type=code) and it seems to primarily affect validating new site creation and new user registration. 

Is this step a necessary or a nice-to-have?

Reply:
Possibly not, @jazzsequence, any edge cases to consider here? _Today_, our search-replace step looks for the constant as part of determining what method of search-replace to run, but we can make that more nuanced, especially if we're pushing some of the behavior out of "detection mode" and into something more explicit in `pantheon.yml`.

Reply:
I tested this locally and `SUBDOMAIN_INSTALL` only affects new site creation (e.g. what options are presented on the Add New Site page). I'm relatively sure that you could even, theoretically, create a subdirectory site via WP-CLI with `SUBDOMAIN_INSTALL` set to true, since WP-CLI bypasses those screens, but I have not tested. I would consider this a nice-to-have rather than a requirement.